`7.0.1` and `7.0.2` releases include bad dependencies

After the addition of the new documentation via `docz`, [this commit](https://github.com/tannerlinsley/react-table/commit/d2acfc423593389f8f7b974c6f544ecf683f0724#diff-b9cfc7f2cdf78a7f4b91a753d10865a2R102) added `docz` and `gatsby-theme-docz` as `"dependencies"` which makes `npm` needlessly pull them when installing `react-table`.

```sh
npm i react-table

# ...

> gatsby-telemetry@1.2.3 postinstall /dev/my-project/node_modules/gatsby-telemetry
> node src/postinstall.js || true


> gatsby-cli@2.11.4 postinstall /dev/my-project/node_modules/gatsby/node_modules/gatsby-cli
> node scripts/postinstall.js


> gatsby@2.20.10 postinstall /dev/my-project/node_modules/gatsby
> node scripts/postinstall.js

# ...
```

I believe those should be declared as `"devDependencies"`.

----

Also, `7.0.2` seems to have a missing or broken `postinstall` script. 

```sh
npm i react-table@7.0.2

> react-table@7.0.2 postinstall  /dev/my-project/node_modules/react-table
> node ./scripts/postinstall.js || exit 0

internal/modules/cjs/loader.js:985
  throw err;
  ^


Error: Cannot find module ' /dev/my-project/node_modules/react-table/scripts/postinstall.js'
    at Function.Module._resolveFilename (internal/modules/cjs/loader.js:982:15)
    at Function.Module._load (internal/modules/cjs/loader.js:864:27)
    at Function.executeUserEntryPoint [as runMain] (internal/modules/run_main.js:74:12)
    at internal/main/run_main_module.js:18:47 {
  code: 'MODULE_NOT_FOUND',
  requireStack: []
}
```

This is due to the addition of a new `"postinstall"` script which is [not included as part of the `npm` bundle](https://github.com/tannerlinsley/react-table/blob/master/package.json#L35). It should either be removed or added to `"files"`.

--- a/package.json
+++ b/package.json
@@ -39,7 +39,8 @@
     "dist",
     "LICENCE",
     "package.json",
-    "README.md"
+    "README.md",
+    "scripts/"
   ],
   "browserslist": "> 0.25%, not dead",
   "peerDependencies": {

--- a/package.json
+++ b/package.json
@@ -37,8 +37,6 @@
     "src/**/*.js",
     "index.d.ts",
     "dist",
-    "LICENCE",
-    "package.json",
     "README.md",
     "scripts/"
   ],

